spec/models/shf_application_spec.rb

```diff
@@ -358,7 +353,9 @@
 
     it 'uses the company main address' do
 
-      expect(accepted_app.se_mailing_csv_str).to eq AddressExporter.se_mailing_csv_str(accepted_app.company.main_address)
+      expect(accepted_app.se_mailing_csv_str)
+        .to eq AddressExporter
+        .se_mailing_csv_str(accepted_app.companies.last.main_address)
```

Comment:
If this is ok with @thesuss  for now, then I'm ok with this (and the related code that satisfies this), _provided that_ you please change the description to something that makes this explicit so we don't forget to change this in the future.  
Suggestion:
`it 'uses the main address of the last company in the list of companies' do`

Reply:
now:
```
"uses the app's latest-added-company main address"
```